frontend/src/components/layout/Navbar.js

```diff
@@ -0,0 +1,87 @@
+import {
+  AppBar,
+  Toolbar,
+  Typography,
+  Menu,
+  MenuItem,
+  IconButton,
+  Button,
+  Grid,
+  Stack,
+} from "@mui/material";
+import MenuIcon from "@mui/icons-material/Menu";
+import DeleteIcon from "@mui/icons-material/Delete";
+import { useState } from "react";
+
+export default function Navbar() {
+  const [anchorEl, setAnchorEl] = useState(null);
+  const handleClick = (event) => {
+    setAnchorEl(event.currentTarget);
+  };
+  const handleClose = () => {
+    setAnchorEl(null);
+  };
+  const open = Boolean(anchorEl);
```

Comment:
I am confused about this part of the code. Why is a boolean const related to a mutable anchorEL?

Reply:
this is being use in the given example. I would change it to useState to let it make more sense.